repos/fdbt-site/tests/pages/service.test.tsx

```diff
@@ -231,32 +230,31 @@ describe('pages', () => {
                     passengerType: 'Adult',
                     services: [
                         {
-                            id: 11,
+                            id: '11',
                             lineName: '123',
                             lineId: '3h3vb32ik',
                             startDate: '05/02/2020',
-                            description: 'this bus service is 123',
                             origin: 'Manchester',
                             destination: 'Leeds',
-                            serviceCode: 'NW_05_BLAC_123_1',
+                            endDate: '25/02/2021',
                         },
                         {
-                            id: 12,
+                            id: '12',
                             lineName: 'X1',
                             lineId: '3h3vb32ik',
                             startDate: '06/02/2020',
-                            description: 'this bus service is X1',
                             origin: 'Edinburgh',
-                            serviceCode: 'NW_05_BLAC_X1_1',
+                            destination: 'Leeds',
```

Comment:
The fact that all these changed should be setting off alarm bells that we're doing something wrong. We no longer have descripton in the data, and we dont have the serviceCode either. This could (and probably does) break things further down the line.